src/mWater_collate/mWater_pull_collate.Rmd

```diff
@@ -0,0 +1,166 @@
+---
+title: "mWater_pull_collate"
+author: "Sam Struthers"
+date: "`r Sys.Date()`"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(tidyverse)
+
+#grab site metadata
+sampling_sites <- read_csv("data/water_sampling_sites.csv")
+# sort for
+upper_sites <- sampling_sites%>%
+  filter(watershed != "CLP  Mainstem-Fort Collins")%>%
+  mutate(site_code = tolower(site_code))
+```
+
+# API Pull and download raw sensor notes
+
+```{r}
+# 
+creds = yaml::read_yaml("src/mWater_collate/mWater_API.yml")
+api_url = as.character(creds["url"])
+rm(creds)
+#create raw download spreadsheet
+download.file(api_url, destfile = "data/mWater_raw_sensor_notes.csv")
+
+```
+
+
+# Tidying
+
+## ALL NOTES
+```{r}
+#read in file
+all_notes <- read_csv(file = "data/mWater_raw_sensor_notes.csv")%>%
```

Comment:
Which one of these dfs are we hoping to append to the field notes sheet that we already have?

Reply:
sensor_notes
apologies that was not clear